src/sentry/snuba/metrics/utils.py

```diff
@@ -430,7 +430,7 @@ def to_intervals(
 
 def get_num_intervals(
     start: Optional[datetime],
-    end: datetime,
+    end: Optional[datetime],
```

Comment:
As you can see the function this function calls has it as `optional`: https://github.com/getsentry/sentry/blob/f2d0a2852daa363da66902fb8ff668433bafcb22/src/sentry/snuba/metrics/utils.py#L393-L395